### ISSUE TITLE
What to do with this?

### ISSUE THREAD

**Issue by coreyfarrell** [istanbuljs/sample-mocha-compilers#6]
I just noticed this repo exists due to a notification from someone posting a bug.  This still uses babel 6 and the legacy istanbul tool.  Do we fix or deprecate and steer people away?

CC @bcoe @JaKXz

**Comment by JaKXz** [comment]
Yeah I think we can archive this with a note pointing to the website

**Comment by JaKXz** [comment]
Same with this project: https://github.com/istanbuljs/sample-babel-node

**Comment by bcoe** [comment]
👍 make it so, let's do our best to reduce confusion about Istanbul vs., Jest, vs., nyc; honestly I think we should point folks at both nyc and Jest -- both of which use modern Istanbul, and do our best to push people away from using the old Istanbul project.

I reached out to @davglass and @gotwarlost about deprecating the legacy istanbul.
